content/docs/use-cases/sharing-resources-efficiently.md

```diff
@@ -0,0 +1,96 @@
+# Sharing Resources Efficiently
+
+Data science teams need to handle large files, rotate the use of special
+processors, and minimize data transfers. This involves provisioning and managing
+resources such as massive on-prem or cloud data stores and powerful servers,
```

Comment:
I would read this statement not as a problem ... but rather as a ... statement :)

Can we just skip it for simplicity ... or move `Minimizing ...` thing  here, closer and simplify this sentence

"... it might require clusters, some software to manage workloads, etc .."

Reply:
PTAL